Kudu.Services/Deployment/DeploymentController.cs

```diff
@@ -111,8 +111,9 @@ public void Delete(string id)
                             {
                                 CreateDeployment(deployResult, result.Value<string>("details"));
 
-                                deployResult.Url = Request.RequestUri;
-                                deployResult.LogUrl = UriHelper.MakeRelative(Request.RequestUri, "log");
+                                Uri baseUri = UriHelper.MakeRelative(UriHelper.GetBaseUri(Request), Request.RequestUri.AbsolutePath);
```

Comment:
I struggle to understand this line. Why isn't baseUri just `UriHelper.GetBaseUri(Request)`? That's what the method name would imply.